### DISCUSSION TITLE
Using variable in process.run

### DISCUSSION THREAD

**Discussion by gavin2812** [savonet/liquidsoap#3124] [User Support]
Hi all,

I've put lufs into a variable using this:

```
value = ref(0.00)
value := input.lufs()
```

If I `print(value)` it shows me the correct value

I then want to do something like this:
`ignore(process.run("curl https://url?level={value}"))`

But all I get at the endpoint is the word 'value' not the actual value of the variable. I've also tried `{!value}` and `#{value}`

No errors are shown when running the script.

How can I use that variable in the URL? 

Thanks in advance.

As an update, if I use this syntax I also get no errors, but the endpoint is not hit at all:
`ignore(process.run("curl https://url?level=#{value}"))`

**Comment by toots** [comment]
You were close! I think what you want is:
```liquidsoap
ignore(process.run("curl https://url?level=#{!value}"))
```

BTW with `2.2.x` the syntax will become (with backward compatible warning):
```liquidsoap
ignore(process.run("curl https://url?level=#{value()}"))
```

**Reply by gavin2812** [reply]
Ahhh, so close!

Thanks as always @toots
